src/client/search/categories/CategoryFilters.js

```diff
@@ -1,26 +1,104 @@
+                $(`.label.year.league.count.content[data-id="${y}"]`).transition('jiggle'));
+          } else {
+            // otherwise if the tab's title league count was hidden show it and hide the
+            // hidden dropdown's count
+            if ($(`.label.year.league.count.title[data-id="${y}"]`).hasClass('hidden')) {
+              newPromiseChain()
+                .then(() =>
+                  $(`.label.year.league.count.title[data-id="${y}"]`).transition('slide up'))
+                .then(() =>
+                  $(`.label.year.league.count.content[data-id="${y}"]`).transition('slide down'));
+            }
+          }
+        }
 
-  componentDidUpdate() {
-    $('.menu .item').tab();
+        let leagueLength = 0;
+        if (this.props.football.leagueTeamsByYear[tabPath]) {
+          leagueLength = this.props.football.leagueTeamsByYear[tabPath].leagues.length;
+        }
+
+        // only fetches the data for the football year if it hasn't already been fetched
+        return this.props.onClickYearTab(tabPath, leagueLength);
+      },
+    });
   }
 
   render() {
-    const currentYear = `${moment().year()}`;
+    const currentYear = moment().year() - 1;
     let seasonYearTabsContent = [];
-    for (let year in this.props.football.seasonsByYear) {
+    for (let y = currentYear; y >= 2013; y--) {
       const tabContentClassName =
-        year === currentYear ? 'ui bottom attached active tab segment' : 'ui bottom attached tab segment';
+        y === currentYear ? 'ui bottom attached active tab segment' : 'ui bottom attached tab segment';
+
       seasonYearTabsContent.push(
-        <div className={tabContentClassName} data-tab={year}>
-          Year: {year}s content
+        <div className={tabContentClassName} data-tab={y}>
+          <div className="ui grid">
+            <div className="center aligned four wide column">
+              <LeagueCategory leagues={this.props.football.seasonsByYear[y]}
+                currentYear={currentYear} tabYear={y}
+                onClickLeague={this.props.onClickCategoryFilter}
+              />
+            </div>
+            <div className="eight wide column">
+              <div className="ui middle aligned grid">
+                <div className="five wide column">
+                  <TeamCategory teamsByLeague={this.props.football.leagueTeamsByYear[y]}
+                    onClickAddTeam={this.props.onClickCategoryFilter}
+                    onClickSelectTeam={this.props.onClickSelectTeam}
+                  />
+                </div>
+                <div className="eleven wide column">
+                  {
+                    !this.props.football.selectedTeam.isSelected &&
+                    this.props.football.leagueTeamsByYear[y] &&
+                    !this.props.football.leagueTeamsByYear[y].isFetching &&
+                      <span className="ui large purple tag label">
+                        Select a team to add players...
+                      </span>
```

Comment:
I have mixed feelings about this (lines 82 - 89), but can't deny that it's effective